File: Lista2/Exercícios_python/Casa/2.py
a = int(input("Digite um número inteiro entre 0 e 100: "))
c = int(input("Digite um número chave: "))

if a>=0 and a<=100:
    if a>c:
        print("A distrância entre esses números é", a-c)
    else:
        print("A distrância entre esses números é", c-a)
else:
    print("Número inválido")

File: Lista2/Exercícios_C/Casa/2.c
#include <stdio.h>
int main ()

{
    int a, b;
    printf("Digite um número inteiro entre 0 e 100: ");
    scanf("%i", &a);
    printf("Digite um número chave: ");
    scanf("%i", &b);

    if (a>=0 && a<=100)
    {
        if (a>b) 
        {
            printf("A distância entre esses números é %i", a-b);
        }
        else
        {
            printf("A distância entre esses números é %i", b-a);
        }
    }
    else
    {
        printf("Número inválido.");
    }

}
